### PR TITLE
test: ceph_test_objectstore: do not override plugin-dir if not necessary

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
tested at http://pulpito.ceph.com/kchai-2016-06-14_03:49:21-rados-wip-ceph_test_objectstore---basic-mira/

**Comment by tchaikov** [comment]
test passed http://pulpito.ceph.com/kchai-2016-06-14_03:49:21-rados-wip-ceph_test_objectstore---basic-mira/

**Comment by tchaikov** [comment]
@ifed01 could you please take a look? thanks.

File: src/test/objectstore/store_test.cc [hunk 1]
```diff
@@ -4485,6 +4485,13 @@ TEST(DummyTest, ValueParameterizedTestsAreNotSupportedOnThisPlatform) {}
 
 #endif
 
+static bool plugin_exists(const string& compressor_dir)
+{
+  string path(compressor_dir);
+  path += "/libceph_snappy.so";
```
Comment (ifed01) [inline src/test/objectstore/store_test.cc]: Not needed IMO

Comment (ifed01) [inline src/test/objectstore/store_test.cc]: Other plugins will depend on snappy presence that's not good IMHO

Comment (tchaikov) [inline src/test/objectstore/store_test.cc]: @ifed01 how about `glob(3)` to see if any plugin exists in this directory?

Comment (tchaikov) [inline src/test/objectstore/store_test.cc]: or you'd suggest we check for the existence of the `compressor_dir` directory instead?

Comment (ifed01) [inline src/test/objectstore/store_test.cc]: What's about check for non-empty compressor_dir directory?

**Comment by tchaikov** [comment]
changelog 
- rebase against master
- do not set "plugin_dir" if `$CEPH_LIB` does not have any plugin in it.

@ifed01 could you take a look again?

**Comment by ifed01** [comment]
lgtm!

**Pull request merged**